map_gui/src/tools/minimap.rs

```diff
@@ -115,7 +115,7 @@ impl<A: AppLike + 'static, T: MinimapControls<A>> Minimap<A, T> {
                 let rect = Polygon::rectangle(20.0, 8.0);
 
                 let default_batch = GeomBatch::from(vec![(color, rect.clone())]);
-                let hover_batch = GeomBatch::from(vec![(app.cs().hovering, rect)]);
+                let hover_batch = GeomBatch::from(vec![(color.alpha(0.5), rect)]);
```

Comment:
Arguably this could just become a `btn_plain_light_icon` with a new SVG for the slightly different shape. Then the hover state would have the padded hitbox and background highlighted, and we could have a proper disabled state so you can't hover on the currently chosen zoom level. Let me know if you'd prefer that change now!

Reply:
I'm ok with what you have, but indeed an svg rectangle *does* sound like an easier way to get consistent behavior.